test/Chainweb/Test/Pact/PactMultiChainTest.hs

```diff
@@ -71,7 +70,8 @@ testVersion :: ChainwebVersion
 testVersion = slowForkingCpmTestVersion peterson
 
 cid :: ChainId
-cid = someChainId testVersion
+cid = unsafeChainId 9
+    -- several tests in this file expect chain 9
```

Comment:
With the new hashable version the value of `someChainId` changes. This fixes the previous value.